### ISSUE TITLE
[Bug]: Primer Interface not working properly with macOS Sonoma 14.0 and 14.1

### ISSUE THREAD

**Issue by thewellington** [quicksilver/Quicksilver#2986]
### Before submitting your bug report, please confirm you have completed the following steps

- [X] Searched the issue is not already listed in the [Issue List](https://github.com/quicksilver/Quicksilver/issues?q=is%3Aissue) (I've searched it)
- [X] Searched in the [Quicksilver Users' Group](https://groups.google.com/forum/#!forum/blacktree-quicksilver) for questions relating to tech support.
- [X] Downloaded the [latest version of Quicksilver](https://qsapp.com/) for your system and confirmed the issue still persists.

### Bug description

1. Primer interface cannot be dragged by the sideways title bar - it is dragged offset from mouse by nearly a quarter of the screen.
2. After calling up the interface and entering whatever you want in pane 1, press the right arrow to get a submenu to pop up.  this menu pops up to the left of the Primer interface (used to pop up to the right) and is not attached in the way it used to be, and cannot be dragged around.

### Steps to reproduce

1. Install QS
2. Press hot key combo to trigger interface
3. try dragging interface
4. trigger submenu
5. try dragging sub menu to new location

### Expected behavior

I expect the interface to drag under the mouse, not offset from he mouse by such a distance
I expect the submenu to be "attached
 to the primary interface and be to the right of that interface.

### MacOS Version

Other

### Quicksilver Version

QS 2.4.2   macOS 14.0 and 14.1

### Relevant Plugins

Primer

### Screenshots

<img width="2560" alt="Screenshot 2023-11-02 at 12 29 37" src="https://github.com/quicksilver/Quicksilver/assets/904076/add56f97-c820-4885-a8c6-4fd482f37ec4">
<img width="2560" alt="Screenshot 2023-11-02 at 12 29 45" src="https://github.com/quicksilver/Quicksilver/assets/904076/291be66d-d619-409b-9fd7-01fdbbd032c2">

**Comment by pjrobertson** [comment]
Are you still having an issue with this? I have checked the latest version with macOS 15.0, and the primer interface appears to work correctly

**Comment by thewellington** [comment]
I agree, under QS 2.4.3 and macOS 15+ this issue seems to have resolved itself.   Not sure when it happened though.
